components/SiteSearch.js

```diff
@@ -105,7 +105,8 @@ class SiteSearch extends React.Component {
 
     let q = query ? `#${query.replace(/ /g,"-")}` : '#';
     if(type === 'Internal') {
-      Router.push(`/page/${slug}${q}`);
+      // Router.push(`/components/${slug}${q}`);
+      Router.push(`/components/${slug}`);
```

Comment:
Ideally, I should not touch this file as it is suppose to be shared between spectrum document site and spectrum css site. But I have to as `/page/` is hard coded here.